tests/unit/spec/base/send-to-device.js

```diff
@@ -85,8 +86,8 @@ describe('send-to-device.js', function() {
 
         it('should handle success', function() {
 
-            spyOn($, 'post').and.callFake(function () {
-                var d = $.Deferred();
+            spyOn(window.$, 'post').and.callFake(function () {
+                var d = window.$.Deferred();
```

Comment:
Is it okay for the `$` to remain here?

Reply:
For now yeah, until we fix https://github.com/mozilla/bedrock/issues/7673